requirements.txt

```diff
@@ -0,0 +1,9 @@
+Django==2.2.14
+django-cors-headers==3.2.1
+djangorestframework==3.11.0
+requests==2.22.0
+django-ckeditor==5.9.0
+django-debug-toolbar==2.2
+django-haystack==2.8.1
+django-import-export==1.2.0
+django-spaghetti-and-meatballs==0.2.2
```

Comment:
Since dependencies are updated at ```ncs-jss/HTTP_200```'s master, you'll have to merge these new versions +  the additional version updates in those already present ones.
In addition, as you mentioned ```drfdocs``` is removed, you may delete ```requirements/dev.txt``` and only use ```requirements/common.txt```.